app/graphql/mutations/save_account_mutation.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Mutations
+  # GraphQL mutation to update the current user account
+  class SaveAccountMutation
+    def call(user, args, _ctx)
```

Comment:
Well, yes, it's simple to recreate it now, but it creates a target for incompatibilities for every devise update.

I think it's okay to do what you suggest, but then please add a comment about it in this mutation and in the Gemfile at the devise-line, so that we can check again when we update devise.